Week7/app/src/test/java/com/kulloveth/covid19virustracker/di/NetworkModuleTest.kt

```diff
@@ -0,0 +1,61 @@
+package com.kulloveth.covid19virustracker.di
+
+import com.kulloveth.covid19virustracker.api.NewsApiService
+import com.kulloveth.covid19virustracker.api.StatusApiService
+import junit.framework.Assert.assertNotNull
+import okhttp3.OkHttpClient
+import org.junit.After
+import org.junit.Before
+import org.junit.Test
+import org.koin.core.context.startKoin
+import org.koin.core.context.stopKoin
+import org.koin.core.qualifier.named
+import org.koin.test.KoinTest
+import org.koin.test.get
+import org.koin.test.inject
+
+class NetworkModuleTest : KoinTest {
+    val newsApi: NewsApiService by inject()
```

Comment:
Above and beyond writing these tests and incorporating the injection in the tests.